### ISSUE TITLE
lower result than paper

### ISSUE THREAD

**Issue by codingwolfman** [VisionLearningGroup/DA_Detection#14]
cityscape->foggy cityscape 32.48
sim10k -> cityscape 38.3
I use trainval_net_global_local.py  gc=True lc=True , bs=1, epoch=20

**Comment by AmingWu** [comment]
Hello, when you test on the foggy cityscape, do you meet the problem of RPN regression box loss becoming NaN?

**Comment by codingwolfman** [comment]
@AmingWu no, all losses are normal, maybe foggy cityscape's annotation has problem, how did you produce the xml file(voc style)

**Comment by AmingWu** [comment]
I download the dataset from this link (https://github.com/tiancity-NJU/da-faster-rcnn-PyTorch)

**Comment by AmingWu** [comment]
@codingwolfman, would you like to share your cityscape data with me? Thank you.

**Comment by DanZhang123** [comment]
Ｈｅｌｌｏ, when I train on cityscape-->foggy cityscape, I meet the problem of RPN regression box loss becoming NaN,　What should I do?
 Looking forward to your reply, thank you very much!

**Comment by DanZhang123** [comment]
> cityscape->foggy cityscape 32.48
> sim10k -> cityscape 38.3
> I use trainval_net_global_local.py gc=True lc=True , bs=1, epoch=20
Ｈｅｌｌｏ, when I train on cityscape-->foggy cityscape, I meet the problem of RPN regression box loss becoming NaN,　My cityscape and foggy cityscape datasets is directity from this link (https://github.com/tiancity-NJU/da-faster-rcnn-PyTorch). I don＇t know where I made mistakes.
Looking forward to your reply, thank you very much!

**Comment by DanZhang123** [comment]
> I download the dataset from this link (https://github.com/tiancity-NJU/da-faster-rcnn-PyTorch)

Hello. I have encountered  the same problem.  My  datasets is also directly downloaded from the same link(https://github.com/tiancity-NJU/da-faster-rcnn-PyTorch). Have you solved it yet.
